core/google/api/core/gapic_v1/method.py

```diff
@@ -222,14 +220,15 @@ def get_topic(name, timeout=None):
             optional ``timeout`` argument. If ``metadata`` is not ``False``, it
             should accept a ``metadata`` argument.
         default_retry (Optional[google.api.core.Retry]): The default retry
-            strategy. If falsy, the method will not retry by default.
+            strategy. If ``None``, the method will not retry by default.
         default_timeout (Optional[google.api.core.Timeout]): The default
-            timeout strategy. If an int or float is specified, the timeout will
-            always be set to that value.
-        metadata (Union(Mapping[str, str], False)): A dict of metadata keys and
+            timeout strategy. Can also be specified as an int or float. If
+            ``None``, the method will not have timeout specified by default.
+        metadata (Optional(Mapping[str, str])): A dict of metadata keys and
             values. This will be augmented with common ``x-google-api-client``
-            metadata. If ``False``, metadata will not be passed to the function
-            at all.
+            metadata. If ``None``, metadata will not be passed to the function
+            at all, if ``USE_DEFAULT_METADATA`` (the default) then only the
```

Comment:
You **could** use a Sphinx `attr` to refer to the sentinel?

Reply:
Done.